src/functions/uploadToRSIS/application/result-client.ts

```diff
@@ -108,6 +108,8 @@ export const getNextUploadBatch = async (baseUrl: string, interfaceType: Interfa
       resolve(resultList);
     }).catch((err) => {
       const ex = mapHTTPErrorToDomainError(err);
+      // tslint:disable-next-line:max-line-length
+      error(`Failed to get next upload batch for interface ${interfaceType} and batch size ${batchSize} with error: `, err);
```

Comment:
use line break instead of disabling linting..

Reply:
Creating new lines messes up the logs and adds a new line in the error log...

Reply:
Actually tested it and it adds a `\n` to the error message